src/software/thunderscope/robot_diagnostics/drive_and_dribbler_widget.py

```diff
@@ -159,79 +306,127 @@ def setup_dribbler(self, title: str) -> QGroupBox:
 
         return group_box
 
-    def toggle_all(self, enable: bool) -> None:
-        """Disables or enables all sliders and buttons depending on boolean parameter
+    def toggle_control_mode(self, use_direct: bool) -> None:
```

Comment:
It would be better to use the `DriveMode` enum for the `use_direct` param instead of a boolean